docs/ibc-relayer.md

```diff
@@ -0,0 +1,109 @@
+# IBC relaying guide
+
+Babylon uses [IBC](https://ibcprotocol.dev/)
+(Inter-Blockchain Communication protocol) to enable cross-chain
+communication with other Cosmos SDK chains. This guide focuses on the specific
+configurations needed
+when relaying with Babylon, particularly around its unique unbonding period
+mechanism.
```

Comment:
Wdyt we add a small subsection below on what IBC functionalities do Babylon support and what do not? For example

- Babylon supports IBC transfer and IBC-Wasm
- Babylon does not support ICA, ICQ, IBC hooks, IBC middleware yet